src/main/java/com/training/sample/pages/SamplePage.java

```diff
@@ -1,19 +1,33 @@
 package com.training.sample.pages;
```

Comment:
Please do not touch the sample tests and sample pages.
All your web based tests should be created in the package - web/tests/
and all web based pages should go in package - web/pages/